### PR TITLE
get Top Projects function added

### PR DESCRIPTION
**Pull request by hsynterkr**
Hi, 
I added `getTopProjects` which returns top utopian projects.

usage example
````js
const utopian = require('utopian-api');

utopian.getTopProjects({
  limit: 5,
  start_date: '2017-12-08',
  end_date: '2018-01-20',
  sort_by: 'contributions',   // contributions or rewards
  retrieve_by: 'projects',    // projects or contributions
  only_new: false
}).then((data) => {
  console.log(data); 
});
````

### CODE REVIEW

File: api.js [hunk 1]
```diff
@@ -2,7 +2,9 @@ const API_HOST = "https://api.utopian.io/api";
 const ENDPOINT_MODERATORS = API_HOST + '/moderators';
 const ENDPOINT_SPONSORS = API_HOST + '/sponsors';
 const ENDPOINT_POSTS = API_HOST + '/posts';
-const ENDPOINT_STATS = API_HOST + '/stats'
+const ENDPOINT_STATS = API_HOST + '/stats';
+const ENDPOINT_PROJECTS = API_HOST + '/posts/top';
```
Comment (wehmoen) [inline api.js]: This is not the endpoint for Projects. use something like ENDPOINT_POSTS_TOP = ENDPOINT_POSTS + "/top"

**Comment by hsynterkr** [comment]
ENDPOINT fixed.
````js
ENDPOINT_POSTS_TOP = ENDPOINT_POSTS + "/top"
````

**Comment by wehmoen** [comment]
This PR does not follow the utopian coding standards: https://travis-ci.org/utopian-io/utopian-api-npm/builds/331218267

**Comment by hsynterkr** [comment]
Pull request closed.

**Pull request closed without merging**